vnext/Shared/Networking/OriginPolicyHttpFilter.cpp

```diff
@@ -0,0 +1,638 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "OriginPolicyHttpFilter.h"
+
+// React Native Windows
+#include <RuntimeOptions.h>
+#include "WinRTTypes.h"
+
+// Boost Library
+#include <boost/algorithm/string.hpp>
+#include <boost/lexical_cast/try_lexical_convert.hpp>
+
+// Windows API
+#include <winrt/Windows.Web.Http.Headers.h>
+
+// Standard Library
+#include <regex>
+
+using std::set;
+using std::wstring;
```

Comment:
I though we had a style guideline not to use `using` for `std::` types

Reply:
I was not aware this was enforced formally.
If it is, I'll update accordingly.

Reply:
I just remember seeing in a comment in another PR.... Maybe @vmoroz / @NikoAri  can chime in?